XCTestBootstrap/Configuration/FBTestLaunchConfiguration.m

```diff
@@ -27,14 +27,16 @@ - (instancetype)initWithTestBundlePath:(NSString *)testBundlePath applicationLau
   _testHostPath = testHostPath;
   _timeout = timeout;
   _shouldInitializeUITesting = initializeUITesting;
+  _testsToRun = testsToRun;
+  _testsToSkip = testsToSkip;
 
   return self;
 }
 
 + (instancetype)configurationWithTestBundlePath:(NSString *)testBundlePath
 {
   NSParameterAssert(testBundlePath);
-  return [[FBTestLaunchConfiguration alloc] initWithTestBundlePath:testBundlePath applicationLaunchConfiguration:nil testHostPath:nil timeout:0 initializeUITesting:NO];
+  return [[FBTestLaunchConfiguration alloc] initWithTestBundlePath:testBundlePath applicationLaunchConfiguration:nil testHostPath:nil timeout:0 initializeUITesting:NO testsToRun:nil testsToSkip:nil];
```

Comment:
In the original pull request I had it like this btw. You can see it in this commit: https://github.com/plu/FBSimulatorControl/commit/1dab86deb599eef2f17cedb6e1ba1f8955a7040e